config/env_spec_request.go

```diff
@@ -479,33 +485,33 @@ func (e *EnvironmentSpecRequest) GetHTTPRequestTransforms() (transforms HTTPRequ
 	return transforms
 }
 
-// returns true if auth is empty or disabled
-func (e *EnvironmentSpecRequest) meetsAuthenticatationRequirements(auth AuthenticationRequirement) bool {
```

Comment:
This method name also feels very like it should return a boolean, can we rename it to maybe `verifyAuthenticationRequirements`?

Reply:
sure, Done.